source/WorkingPractices/TestSuites/lfric_core.rst

```diff
@@ -3,11 +3,12 @@
 Testing LFRic Core
 ==================
 
-LFRic testing is run with the following command from a working copy:
+LFRic testing is now Cylc8 only and launched with rose-stem commands:
```

Comment:
To make this slightly less of the moment I'd suggest just including the "export CYLC_VERSION=8" in the command and use this headline sentence to just say that LFRic Core uses rose-stem (like Apps, UM etc). 

It might then be worth a "note" box to say that calling make test-suite was deprecated from the core2.0 revision or something like that?

Reply:
Done